samples/photo_album/config/application.rb

```diff
@@ -4,7 +4,7 @@
 require "active_record/railtie"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-require "active_resource/railtie"
+# require "active_resource/railtie"
```

Comment:
?

Reply:
This module was removed from rails long ago and our sample project does not use it.